github/bump_versions/bump_versions.rb

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env ruby
+
+# This script bumps the version in all YaST packages
+#
+# You need admin access rights to temporarily disable the GitHub branch protection
+# and allow direct push to "master".
+#
+
+# the GitHub organization
+GH_ORG = "yast".freeze
+
+# install missing gems
+if !File.exist?("./.vendor")
+  puts "Installing the needed Rubygems to ./.vendor/bundle ..."
+  system "bundle install --path .vendor/bundle"
+end
+
+require "rubygems"
+require "bundler/setup"
+
+require "octokit"
+
+DRY_RUN = ENV["DRY_RUN"]
+
+# new package version
+NEW_PACKAGE_VERSION = "4.4.0".freeze
```

Comment:
it would be nice to have it as argument, but fine for one time in a while script.